e2e-tests/portal/cypress/e2e/journeys/payment-report-officer/utilisation-report-upload/confirm-and-send.spec.js

```diff
@@ -5,32 +5,43 @@ const relativeURL = require('../../../relativeURL');
 const { BANK1_PAYMENT_REPORT_OFFICER1 } = MOCK_USERS;
 
 context('Confirm and send', () => {
-  beforeEach(() => {
-    cy.login(BANK1_PAYMENT_REPORT_OFFICER1);
-    cy.visit(relativeURL('/utilisation-report-upload'));
+  describe('After logging in and submitting a valid file', () => {
+    beforeEach(() => {
+      cy.login(BANK1_PAYMENT_REPORT_OFFICER1);
+      cy.visit(relativeURL('/utilisation-report-upload'));
 
-    utilisationReportUpload.utilisationReportFileInput().attachFile('valid-utilisation-report.xlsx');
-    utilisationReportUpload.continueButton().click();
-  });
+      utilisationReportUpload.utilisationReportFileInput().attachFile('valid-utilisation-report.xlsx');
+      utilisationReportUpload.continueButton().click();
+    });
 
-  it('Should route to the Upload Report page when the back button is selected', () => {
-    confirmAndSend.backLink().click();
+    it('Should route to the Upload Report page when the back button is selected', () => {
+      confirmAndSend.backLink().click();
 
-    confirmAndSend.mainHeading().should('not.exist');
-    confirmAndSend.currentUrl().should('contain', '/utilisation-report-upload');
-  });
+      confirmAndSend.mainHeading().should('not.exist');
+      confirmAndSend.currentUrl().should('contain', '/utilisation-report-upload');
+    });
+
+    it('Should route to the Upload Report page when the change button is selected', () => {
+      confirmAndSend.changeLink().click();
+
+      confirmAndSend.mainHeading().should('not.exist');
+      confirmAndSend.currentUrl().should('contain', '/utilisation-report-upload');
+    });
 
-  it('Should route to the Upload Report page when the change button is selected', () => {
-    confirmAndSend.changeLink().click();
+    it('Should route to the Confirmation page when the Confirm and Send button is selected', () => {
+      confirmAndSend.confirmAndSendButton().click();
 
-    confirmAndSend.mainHeading().should('not.exist');
-    confirmAndSend.currentUrl().should('contain', '/utilisation-report-upload');
+      confirmAndSend.mainHeading().should('not.exist');
+      confirmAndSend.currentUrl().should('contain', '/confirmation');
+    });
   });
 
-  it('Should route to the Confirmation page when the Confirm and Send button is selected', () => {
-    confirmAndSend.confirmAndSendButton().click();
+  describe('After logging in but not submitting a file', () => {
+    it('Should route to the Upload Report page when you try and access the confirm and send page directly', () => {
```

Comment:
Need to check if the express session will still exist at this point with a fresh login or if I'll need to force a logout first to destroy the session